docs/getting-started-guides/windows/sample-cniconfig.json

```diff
@@ -0,0 +1,49 @@
+{
+    "cniVersion":  "0.2.0",
+    "name":  "l2bridge",
+    "type":  "wincni.exe",
+    "master":  "Ethernet",
+    "ipam":  {
+                 "environment":  "azure",
+                 "subnet":  "10.10.187.64/26",
+                 "routes":  [
+                                {
+                                    "GW":  "10.10.187.66"
+                                }
+                            ]
+             },
+    "dns":  {
+                "Nameservers":  [
+                                    "11.0.0.10"
+                                ]
+            },
+    "AdditionalArgs":  [
+                           {
+                               "Name":  "EndpointPolicy",
+                               "Value":  {
+                                             "Type":  "OutBoundNAT",
+                                             "ExceptionList":  [
+                                                                   "11.0.0.0/8",
+                                                                   "10.10.0.0/16",
+                                                                   "10.127.132.128/25"
+                                                               ]
+                                         }
+                           },
+                           {
+                               "Name":  "EndpointPolicy",
+                               "Value":  {
+                                             "Type":  "ROUTE",
+                                             "DestinationPrefix":  "11.0.0.0/8",
+                                             "NeedEncap":  true
+                                         }
+                           },
+                           {
+                               "Name":  "EndpointPolicy",
+                               "Value":  {
+                                             "Type":  "ROUTE",
+                                             "DestinationPrefix":  "10.127.132.213/32",
+                                             "NeedEncap":  true
+                                         }
+                           }
+                       ]
+}
```

Comment:
This file should contain in the filename the network topology for which it applies. The filename suggests that this is valid for all CNI plugins.

Reply:
renamed to sample-l2bridge-wincni-config.json